### PR TITLE
Add poly example integration test for openpbs driver

### PR DESCRIPTION
**Pull request by jonathan-eq**

### CODE REVIEW

**Comment by jonathan-eq** [comment]
I didnt have `QUEUE_OPTION TORQUE MAX_RUNNING 50` set

**Comment by jonathan-eq** [comment]
Added reasonable timeout of 35s. OpenPBS driver is flaky.

**Comment by jonathan-eq** [comment]
Increased timeout to 180s. If it reaches this, it should be hanging for sure.

**Comment by jonathan-eq** [comment]
I am considering this PR as complete. It adds a test to prove an issue in the openpbs driver.

**Comment by berland** [comment]
This test passes when #7165 is merged.

File: tests/integration_tests/scheduler/test_openpbs_driver.py [hunk 1]
```diff
@@ -94,3 +100,28 @@ async def finished(iens, returncode, aborted):
...
+@pytest.mark.timeout(180)
+@pytest.mark.integration_test
+def test_openpbs_driver_with_poly_example(tmp_path, source_root, monkeypatch):
+    shutil.copytree(
+        os.path.join(source_root, "test-data", "poly_example"),
+        os.path.join(str(tmp_path), "poly_example"),
+    )
+    monkeypatch.chdir(tmp_path)
+    with open("poly_example/poly.ert", mode="a+", encoding="utf-8") as f:
+        f.write("QUEUE_SYSTEM TORQUE\nQUEUE_OPTION TORQUE MAX_RUNNING 50")
```
Comment (berland) [inline tests/integration_tests/scheduler/test_openpbs_driver.py:114]: I would say 2 realization should suffice.

Comment (berland) [reply tests/integration_tests/scheduler/test_openpbs_driver.py:114]: (that will render setting MAX_RUNNING irrelevant)

**Review by berland — APPROVED** [review]

**Pull request merged**